mpas_analysis/config.default

```diff
@@ -734,7 +734,7 @@ contourLevelsDifference = np.arange(-5., 6., 2.)
 ## options related to plotting the Global MOC
```

Comment:
How about we also fix the typo above (Global--> IndoPacific)?

Reply:
Done!